src/autocomplete.js

```diff
@@ -600,7 +600,7 @@ export default class Autocomplete extends Component {
           >
             {selectedOptions.map((option, index) => {
               return (
-                <li id={`${id}__option--${index}`} className='autocomplete__selected-option'>
+                <li id={`${id}__selected-option--${index}`} className='autocomplete__selected-option'>
```

Comment:
May be worth writing a test so that this doesn't happen again in a future regression?

Along the lines of: `expect(document.querySelector("#foo__option--1").length).toEqual(1)`

Reply:
I think this still stands ^ could you let me know what you think Alex? You've tested counting elements of a certain class, but this test would be to do with IDs. It's only the duplicate IDs that would cause a11y issues.